src/ukify/ukify.py

```diff
@@ -69,7 +69,10 @@ def path_is_readable(s: str | None) -> pathlib.Path | None:
     if s is None:
         return None
     p = pathlib.Path(s)
-    p.open().close()
+    try:
+        p.open().close()
+    except IsADirectoryError:
+        pass
```

Comment:
this really should use access(), this open/close mess with handling dirs is just a mess...